src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimantResponseService.java

```diff
@@ -52,14 +54,21 @@ public void save(
 
         Claim updatedClaim = caseRepository.saveClaimantResponse(claim, response, authorization);
 
-        if (ACCEPTATION == response.getType()) {
-            formaliseResponseAcceptanceService.formalise(updatedClaim, (ResponseAcceptation) response, authorization);
-        }
+        formaliseResponseAcceptance(response, updatedClaim, authorization);
 
         eventProducer.createClaimantResponseEvent(updatedClaim);
         appInsights.trackEvent(getAppInsightsEvent(response), claim.getReferenceNumber());
     }
 
+    private void formaliseResponseAcceptance(ClaimantResponse claimantResponse, Claim claim, String authorization) {
```

Comment:
Since the claim passed to this method is updated with claimantresponse. cant we fetch the claimantresponse from the claim itself? why do we need to pass it as an argument. Happy to be corrected.

Reply:
Agree. previously the claim object did't have `ClaimantResponse` at this point, hence we were passing these separately. Now we can clean this up as `updatedClaim` contains the `ClaimantResponse` preserved in database.

@ellie-harrison This can be a separate PR to cleanup.